diff --git a/gettext-runtime/gnulib-lib/memmove.c b/gettext-runtime/gnulib-lib/memmove.c
--- a/gettext-runtime/gnulib-lib/memmove.c
+++ b/gettext-runtime/gnulib-lib/memmove.c
@@ -11,8 +11,8 @@
-memmove (void *dest0, void const *source0, size_t length)
+memmove_gnu (void *dest0, void const *source0, size_t length)
 {
   char *dest = dest0;
   char const *source = source0;
   if (source < dest)
     /* Moving from low mem to hi mem; start at end.  */
     for (source += length, dest += length; length; --length)
       *--dest = *--source;
diff --git a/gettext-runtime/gnulib-lib/memchr.c b/gettext-runtime/gnulib-lib/memchr.c
--- a/gettext-runtime/gnulib-lib/memchr.c
+++ b/gettext-runtime/gnulib-lib/memchr.c
@@ -58,9 +58,9 @@
 void *
-__memchr (void const *s, int c_in, size_t n)
+__memchr_gnu (void const *s, int c_in, size_t n)
 {
   /* On 32-bit hardware, choosing longword to be a 32-bit unsigned
      long instead of a 64-bit uintmax_t tends to give better
      performance.  On 64-bit hardware, unsigned long is generally 64
      bits already.  Change this typedef to experiment with
      performance.  */
   typedef unsigned long int longword;
diff --git a/gettext-tools/libgettextpo/memchr.c b/gettext-tools/libgettextpo/memchr.c
--- a/gettext-tools/libgettextpo/memchr.c
+++ b/gettext-tools/libgettextpo/memchr.c
@@ -58,9 +58,9 @@
 void *
-__memchr (void const *s, int c_in, size_t n)
+__memchr_gnu (void const *s, int c_in, size_t n)
 {
   /* On 32-bit hardware, choosing longword to be a 32-bit unsigned
      long instead of a 64-bit uintmax_t tends to give better
      performance.  On 64-bit hardware, unsigned long is generally 64
      bits already.  Change this typedef to experiment with
      performance.  */
   typedef unsigned long int longword;
-- 
